src/main/java/de/tum/in/www1/artemis/web/rest/PrivacyStatementResource.java

```diff
@@ -29,12 +29,12 @@ public PrivacyStatementResource(PrivacyStatementService privacyStatementService)
      * @param language the language of the privacy statement
      * @return the privacy statement
      */
-    @GetMapping("/privacy-statement")
+    @GetMapping("privacy-statement")
     public PrivacyStatement getPrivacyStatement(@RequestParam("language") String language) {
```

Comment:
```suggestion
    @GetMapping("privacy-statement")
    @PreAuthorize("permitAll()")   
    public PrivacyStatement getPrivacyStatement(@RequestParam("language") String language) {
```

Reply:
doesn't work for the unauthenticated user because of a missing security context.